### PR TITLE
[LYN-5564] Update AWSNativeSDK revision

### PR DESCRIPTION
**Pull request by onecent1101**
# Details
Packages have been promoted to prod, see jira SPEC-8102 and SPEC-8105

# Testing
Get the same failures as nightly build, the failures are caused by dev branch.
Windows:
https://jenkins-pipeline.agscollab.com/blue/organizations/jenkins/O3DE-LY-FORK/detail/LYN-5564/6/pipeline

Linux:
https://jenkins-pipeline.agscollab.com/blue/organizations/jenkins/O3DE-LY-FORK/detail/LYN-5564/5/pipeline

Mac:
https://jenkins-pipeline.agscollab.com/blue/organizations/jenkins/O3DE-LY-FORK/detail/LYN-5564/4/pipeline

Signed-off-by: onecent1101 <liug@amazon.com>

### CODE REVIEW

**Review by junbo75 — APPROVED** [review]

**Comment by lawsonamzn** [comment]
looks good to me, once we have enough approvals we can schedule an AR build.

**Review by lawsonamzn — APPROVED** [review]

**Pull request merged**
